plugins/postcss-rgb-mapping/README.md

```diff
@@ -32,5 +42,21 @@ Into this:
 		var(--spectrum-seafoam-200-rgb),
 		var(--spectrum-seafoam-200-opacity)
 	);
+	--spectrum-transparent-white-100-rgb: 100, 100, 100;
+	--spectrum-transparent-white-100-opacity: 0.5;
+	--spectrum-transparent-white-100: rgba(
+		var(--spectrum-transparent-white-100-rgb),
+		var(--spectrum-transparent-white-100-opacity)
+	);
+	--disabled-static-white-background-color-rgb: var(
+		--spectrum-transparent-white-100-rgb
+	);
+	--disabled-static-white-background-color-opacity: var(
+		--spectrum-transparent-white-100-opacity
+	);
+	--disabled-static-white-background-color: rgba(
+		var(--disabled-static-white-background-color-rgb),
+		var(--disabled-static-white-background-color-opacity)
+	);
```

Comment:
I wonder if separating the two chunks of example code with CSS comments (e.g. `/* Example 2: Remapping of transparent tokens */` and `/* Example 1: RGB and opacity postfixed variables */`) explaining what it does would be helpful in trying to see the changes. Since the before/after code is a lot longer now, it's a little less clear trying to compare and look for differences. Just a thought.

Reply:
I like that idea. I feel like I struggled a little bit to be concise with the CSS comments. Let me know if I can clarify anything.

Reply:
Perfect, thanks!